crates/types/src/audio.rs

```diff
@@ -0,0 +1,80 @@
+#[derive(Copy, Clone, Debug)]
+pub enum SpeakerRequest {
+    PlaySound { sound: Sound },
+}
+
+macro_rules! define_sound_enum {
+    ($($name:ident,)*) => {
+        #[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
+        pub enum Sound {
+            $(
+                $name,
+            )*
+        }
+
+        impl Sound {
+            pub fn all() -> &'static [Self] {
+                &[
+                    $(
+                        Self::$name,
+                    )*
+                ]
+            }
+        }
+    };
+}
```

Comment:
I thought about using `enum-iterator` in the beginning but decided against it because this simple macro would be all I need and I'm not adding another to our 700-ish dependencies. But yeah, I'm using it now. :man_shrugging: We should really try to clean our dependencies for better compilation speed. 600-700 compilation targets is a lot...